app/models/EpicTests.scala

```diff
@@ -42,7 +42,8 @@ case class EpicVariant(
   footer: Option[String] = None,
   showTicker: Boolean = false,
   backgroundImageUrl: Option[String] = None,
-  cta: Option[Cta]
+  cta: Option[Cta],
+  classNames: List[String] = Nil
```

Comment:
this change is backwards-compatible with our existing data because, although the field is not optional, it defaults to `Nil` if not present